Code-4-Community/speak-for-the-trees-frontend
# Sk.increase log in text bkg contrast

## Checklist

- [x] 1. Run `npm run check`
- [x] 2. Run `npm run test`
- [x] 3. Change ClickUp ticket status to "In Review"
- [x] 4. After making the PR, add PR link to ClickUp ticket

## Why
[ClickUp Ticket](https://app.clickup.com/t/36gw0h9)
This cosmetic change is to make the login and signup page text darker. The goal is to improve the make this text easier to read with more contrast between the text and background colors. 

## This PR
This change adds a color with rgb values (100, 100, 100) and hex code #646464 to colors.tsx. The login and sign up page footers were then adjusted to use this new color value.

## Screenshots

|||||
| --- | --- | --- | --- |
|Sign up before|Sign up after|Login before|Login After|
| <img width="300" alt="Screen Shot 2023-04-01 at 4 51 19 PM" src="https://user-images.githubusercontent.com/30560773/229313515-8220cca5-e62e-473a-96c3-de1cae4b8cb5.png"> | <img width="300" alt="Screen Shot 2023-04-01 at 4 48 39 PM" src="https://user-images.githubusercontent.com/30560773/229313523-bdf674f8-4fed-4fca-9eb9-71f0a66923a5.png"> | <img width="300" alt="Screen Shot 2023-04-01 at 4 51 24 PM" src="https://user-images.githubusercontent.com/30560773/229313532-9772ce8e-fbb4-4dbf-8004-61ba9207d4d7.png"> | <img width="300" alt="Screen Shot 2023-04-01 at 4 48 53 PM" src="https://user-images.githubusercontent.com/30560773/229313539-31836c6b-13a3-458e-8b8b-6d814a73ea7b.png"> |

## Verification Steps

In order to verify this PR, I manually checked that the text color on the sign in and login pages was as expected
